script/Arcueid/Day 7/07_00_ARC07_3D.txt

```diff
@@ -306,7 +310,7 @@ Because she said that killing vampires was her objective,
 [sha:ea680f193b86d7a6a2f81178714229b147f0385d]{
 -- Page 252, Offset 17181.
 -- 膝から崩れ落ちそうな脱力。
-Strength drains out of me, and I feel as though I'm about to collapse to my knees.
+Strength drains out of me, and I feel as though I'm about to fall to my knees.
```

Comment:
The strength drains from my body, and I feel like I'm about to fall to my knees.